### ISSUE TITLE
HeidiSQL doesnt show view Editor

### ISSUE THREAD

**Issue by reitbrud** [HeidiSQL/HeidiSQL#1816]
**Preconditions**

* HeidiSQL version: 12.5
* Database type and version: mariadb 10.5.19
* OS: Windows 11

**Describe the bug**

**To Reproduce**

**Screenshots or Crash reports**

![Screenshot 2023-05-24 205319](https://github.com/HeidiSQL/HeidiSQL/assets/34591964/1c51bebc-1a87-4e48-9d6d-ae451437d7f6)

**Comment by ansgarbecker** [comment]
Confirmed, although I can at least see the code editor, but it seems it gets the wrong dimensions because it is not visible immediately after loading. Does not happen when I'm on the "Table" tab before.

![grafik](https://github.com/HeidiSQL/HeidiSQL/assets/7986591/3dd9bde3-a88e-46df-aeba-a06646f75b48)

Will check that.

**Comment by ansgarbecker** [comment]
Oh and that also does not happen if the screen is at 100% DPI. Only my first screen at 125% DPI has that "shrink" effect. Can you confirm it works normal with 100% DPI?

**Comment by reitbrud** [comment]
Yes i can confirm that, if i change scaling from 125% to 100% it is working all correct.

If i have full screen and 125% scaling i have no view editor.
If i have not full screen and 125% scaling i have smaller view editor, like image from you.
If i have full screen or not full screen and 100% scaling factor all works well.
